launch/client.py

```diff
@@ -492,6 +509,18 @@ def edit_model_endpoint(
         )  # Returned from server as "creation"
         logger.info("Endpoint edit task id is %s", endpoint_creation_task_id)
 
+
+    def get_model_endpoint(self, endpoint_name: str) -> Optional[Union[AsyncEndpoint, SyncEndpoint]]:
+        try:
+            resp = self.connection.get(os.path.join(ENDPOINT_PATH, endpoint_name))
+        except APIError:
+            logger.exception("Got an error when retrieving endpoint %s", endpoint_name)
+            return None
+
+        # TODO: Return endpoint type and create the right type
```

Comment:
guessing this is still in progress? or does it require adding some stuff to the Gateway

Reply:
Oh yeah I think I'll just have the Gateway return the `endpoint_type`, will do that in the other PR.

Reply:
should we remove the TODO now?